LifeSpace/Debug/LogManager.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  LogStore.swift
+//  LifeSpace
+//
+//  Created by Vishnu Ravi on 11/8/24.
+//
+
+import Foundation
+import OSLog
+import Spezi
+import SwiftUI
+
+actor LogManager: Module, DefaultInitializable, EnvironmentAccessible {
+    @Application(\.logger) private var logger
+    
+    func query(
+        startDate: Date? = nil,
+        endDate: Date? = nil,
```

Comment:
Wondering it it might make sense to enforce passing in a `Range` here; a start position since the lastest boot might result in unintentionally quite long queries. It could be an open range with no end date.